examples/swift/hello_world/ViewController.swift

```diff
@@ -49,7 +49,9 @@ final class ViewController: UITableViewController {
     let requestID = self.requestCount
     let request = RequestBuilder(method: .get, scheme: kRequestScheme,
                                  authority: kRequestAuthority,
-                                 path: kRequestPath).build()
+                                 path: kRequestPath, upstreamHttpProtocol: .http2)
+        .addUpstreamHttpProtocol(.http2)
```

Comment:
Can we update objc/java too?

Reply:
I intentionally left this with h1 so we have coverage for both paths. I can drop a comment in the examples to make this explicit.

Reply:
Comment would be nice 👍